terraform/aws/components/nhais-responder/locals_env_variables.tf

```diff
@@ -0,0 +1,64 @@
+locals {
+  environment_variables = concat(var.nhais_environment_variables,[
+    {
+      name  = "NHAIS-RESPONDER_OUTBOUND_SERVER_PORT"
```

Comment:
`NHAIS-RESPONDER_...` are not environment variables the application understands. This need to be `NHAIS_...` - same as for the nhais adaptor.

Reply:
Fixed